provider/go.mod

```diff
@@ -6,8 +6,8 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.33.0
 	github.com/hashicorp/terraform-provider-google-beta v0.0.0
 	github.com/pulumi/providertest v0.0.11
-	github.com/pulumi/pulumi-terraform-bridge/pf v0.34.0
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.81.0
+	github.com/pulumi/pulumi-terraform-bridge/pf v0.33.1-0.20240425153425-3770ed5a7ac4
```

Comment:
The pf version seems to be reverted?

Reply:
It seems to be, but it isn't. This is because the latest PF release (v0.34.0) commit didn't land in master, so isn't a parent of the current bridge HEAD.

Reply:
ah, it's the same sha as the bridge package. Kind of unfortunate but harmless